Hi3Helper.Core/Lang/en_US.json

```diff
@@ -946,7 +962,10 @@
     "DownloadSettingsOption1": "Start game after install",
 
     "OpenInExternalBrowser": "Open in External Browser",
-    "CloseOverlay": "Close Overlay"
+    "CloseOverlay": "Close Overlay",
+    
+    "DbGenerateUid_Title": "Are you sure to change your user ID?",
+    "DbGenerateUid_Content": "Changing user ID will cause your data associated with current ID to be lost if you lose your current user ID."

```

Comment:
`Changing the current user ID will cause the associated data to be lost if you lose it.`

Reply:
Fixed